conanfile.py

```diff
@@ -1,26 +1,25 @@
 from conans import ConanFile, CMake
 
 class libiconvConan(ConanFile):
-    name = "libiconv"
-    version = "1.15.0"
-    branch = "master"
-    generators = "cmake"
-    settings =  "os", "compiler", "arch", "build_type"
-    options = {"shared": [True, False], "fPIC": [True, False]}
-    default_options = "shared=False", "fPIC=True"
-    url = "http://github.com/kwallner/libiconv"
-    scm = { "type": "git", "url": "auto", "revision": "auto" }
-    no_copy_source = True
-    
-    def config_options(self):
-        del self.settings.compiler.libcxx
+    name = 'libiconv'
+    lib_version = '1.15.0'
+    revision = '0'
+    version = '{}-{}'.format(lib_version, revision)
+    settings = 'os', 'compiler', 'build_type', 'arch'
+    description = 'Convert text to and from Unicode'
+    url = 'https://github.com/Pix4D/libiconv'
+    license = 'LGPL-3.0'
+    generators = 'cmake'
+    exports_sources = '*'
+    options = {'shared': [True, False]}
+    default_options = { 'shared': True }
 
     def build(self):
-        cmake = CMake(self)
+        cmake = CMake(self, parallel=True)
+        cmake.definitions['BUILD_SHARED_LIBS'] = self.options.shared
```

Comment:
[info, no need to check anything] Note: this is automagically done by the CMake constructor above.